<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line splitter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>1080</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1080</width>
     <height>380</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="firstLine">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>15</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondLine">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>55</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="firstL1">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>214</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondL1">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>254</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="firstL2">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>294</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondL2">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>334</y>
      <width>900</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="egArea">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>95</y>
      <width>900</width>
      <height>95</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>95</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>95</height>
     </size>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>898</width>
       <height>93</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>193</y>
      <width>500</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Select the column where the new example begins by clicking on its number.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>22</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>First half (line)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>62</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Second half (line)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>221</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>First half (L1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>301</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>First half (L2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>261</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Second half (L1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>341</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>Second half (L2)</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>901</x>
     <y>398</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
